src/plugins/intel_gpu/include/intel_gpu/runtime/device_info.hpp

```diff
@@ -29,6 +29,12 @@ struct device_uuid {
     std::array<uint8_t, max_uuid_size> val;
 };
 
+/// @brief Structure to represent gpu device LUID
+struct device_luid {
+    static const constexpr size_t max_luid_size = 8;
+    std::array<uint8_t, max_luid_size> val;
+};
```

Comment:
why do we need one more new definition in GPU plugin? why public one cannot be used?

Reply:
Removed it for both luid and uuid